code/modules/mob/living/simple_animal/friendly/pet.dm

```diff
@@ -9,11 +9,15 @@
 
 /mob/living/simple_animal/pet/attackby(obj/item/O, mob/user, params)
 	if(istype(O, /obj/item/newspaper))
-		if(!stat)
-			user.visible_message("<span class='notice'>[user] baps [name] on the nose with the rolled up [O].</span>")
-			spawn(0)
-				for(var/i in list(1,2,4,8,4,2,1,2))
-					setDir(i)
-					sleep(1)
+		var/obj/item/newspaper/N = O  // Cast O to a newspaper object
```

Comment:
This reads like an unintended inclusion, please pull this out into a separate PR!